src/ol/interaction/Snap.js

```diff
@@ -381,7 +381,8 @@ class Snap extends PointerInteraction {
     if (currentMap) {
       keys.forEach(unlistenByKey);
       keys.length = 0;
-      features.forEach(this.forEachFeatureRemove_.bind(this));
```

Comment:
The `forEachFeatureRemove_` method can now be removed - this was the only call site.

Reply:
The` forEachFeatureRemove_` method has been removed